oxidation/libparsec/crates/types/build.rs

```diff
@@ -0,0 +1,10 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use glob::glob;
+
+fn main() {
+    for entry in glob("schema/**/*.json").expect("Failed to parse glob pattern") {
```

Comment:
`**/*.json` is it equivalent to `**.json` or does it means we must be in a subfolder ?